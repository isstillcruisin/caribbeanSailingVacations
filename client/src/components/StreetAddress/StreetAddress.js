import React from "react";

import {Row, Container} from 'react-bootstrap';

const StreetAddress = props => {
  console.log("****", props);
  return (
    <Container>
      <Row className='footer-address'>
        <i>{props.address.name}</i>
        <i>{props.address.companyName}</i>
        <i>{props.address.streetAddress}</i>
        <i>{props.address.city} {props.address.state} {props.address.country} {props.address.zipCode}</i>
      </Row>
    </Container>
  )
};

export default StreetAddress;
