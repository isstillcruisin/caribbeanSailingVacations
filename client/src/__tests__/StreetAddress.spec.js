import React from "react";
import { create } from "react-test-renderer";
import StreetAddress from '../components/StreetAddress';

describe("StreetAddress component", () => {
  test("Matches the snapshot", () => {
    const address = create(<StreetAddress 
      address={{
        name: 'test',
        companyName: 'test company',
        streetAddress: '1 Tester Lane',
        city: 'Test',
        state: 'Tt',
        country: 'Test USA',
        zipCode: '00000-0000'
      }}
    />);
    expect(address.toJSON()).toMatchSnapshot();
  });
});