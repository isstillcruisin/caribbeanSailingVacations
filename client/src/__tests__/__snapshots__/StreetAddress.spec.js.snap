// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StreetAddress component Matches the snapshot 1`] = `
<div
  className="container"
>
  <div
    className="footer-address row"
  >
    <i>
      test
    </i>
    <i>
      test company
    </i>
    <i>
      (000)000-0000
    </i>
    <i>
      1 Tester Lane
    </i>
    <i>
      Test
       
      Tt
       
      Test USA
       
      00000-0000
    </i>
  </div>
</div>
`;
